lightly/transforms/dino_transform.py

```diff
@@ -15,6 +15,21 @@
 class DINOTransform(MultiViewTransform):
     """Implements the global and local view augmentations for DINO [0].
 
+    Input to this transform:
+        PIL Image or Tensor.
+
+    Output of this transform:
+        List of Tensor of length 2x global + n_local_views. (8 by default)
```

Comment:
Can you make a suggestion of how this would look like for one of the models?

Reply:
```
    def __call__(self, image: Union[Tensor, Image]) -> List[Tensor]:
        """Transforms an image into multiple views.

        Args:
            image:
                PIL image or tensor.

        Returns:
            List of tensors with length 2 * global + n_local_views. (8 by default).

        """
        return super()(image)
```

Reply:
Great, thanks!

I think we can do both. Because the way the current docs page works we show all the attributes as well. 
![image](https://github.com/lightly-ai/lightly/assets/1522766/6915de1a-ad3e-4a96-85cc-f4e3f5d2c3c9)


And in order to see the `__call__` method one would have to scroll down a lot. Maybe we can even add the `__call__` method in a second step as it's mostly use for type hints?

The main motivation for my changes was that I want to quickly see which input/output a transform is using and what augmentations are applied.

Reply:
I did not add the `__call__` methods and their docstrings @guarin. When we looked at the example code it looked like the typhints already worked. We can always add them at a later stage in case we run into some issues.